hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/TestContainerReplication.java

```diff
@@ -150,6 +154,8 @@ public void testContainerReplication() throws Exception {
         -> destinationDatanodeDatanodeStateMachine.getSupervisor()
         .getReplicationRequestCount() > 0, 1000, 20_000);
 
+    Assert.assertEquals(1, destinationDatanodeDatanodeStateMachine.
+            getSupervisor().getReplicationSuccessCount());
```

Comment:
Is there any guarantee that if we have a request (ie. requestCount > 0), then it is also successfully completed?  I wouldn't think so.

I think we should wait for success:

```suggestion
        .getReplicationSuccessCount() > 0, 1000, 20_000);
```